test/multi-ns/connection_test.go

```diff
@@ -0,0 +1,155 @@
+package multi_ns_test
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	clientset *kubernetes.Clientset
+	clientDep = &appsv1.Deployment{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "simple-client",
+			Namespace: "default",
+		},
+		Spec: appsv1.DeploymentSpec{
+			Replicas: int32Ptr(1),
```

Comment:
Instead of having this function you could do something like:

```
// Create it like this
replicas := int32(1)

// use it like this:
Replicas: &replicas
```

Reply:
I created this helper function because I want to set the replica number in one line for better readability. Is there any benefits of using a helper variable rather than a helper function?

Reply:
Not necessarily, I was suggesting an alternative.